aptos-move/aptos-vm/src/counters.rs

```diff
@@ -3,20 +3,24 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use aptos_metrics_core::{
-    exponential_buckets, register_histogram, register_int_counter, register_int_counter_vec,
-    register_int_gauge, Histogram, IntCounter, IntCounterVec, IntGauge,
+    exponential_buckets, histogram_opts, register_histogram, register_int_counter,
+    register_int_counter_vec, register_int_gauge, Histogram, IntCounter, IntCounterVec, IntGauge,
 };
 use once_cell::sync::Lazy;
 
+const BLOCK_EXECUTION_TIME_BUCKETS: [f64; 16] = [
+    0.20, 0.30, 0.40, 0.50, 0.60, 0.70, 0.80, 0.90, 1.0, 1.25, 1.5, 1.75, 2.0, 3.0, 4.0, 5.0,
+];
+
 pub static BLOCK_EXECUTOR_EXECUTE_BLOCK_SECONDS: Lazy<Histogram> = Lazy::new(|| {
-    register_histogram!(
+    let histogram_opts = histogram_opts!(
         // metric name
         "block_executor_execute_block_seconds",
         // metric description
         "The time spent in seconds for executing a block in block executor",
-        exponential_buckets(/*start=*/ 1e-3, /*factor=*/ 2.0, /*count=*/ 20).unwrap(),
-    )
-    .unwrap()
+        BLOCK_EXECUTION_TIME_BUCKETS.to_vec()
+    );
+    register_histogram!(histogram_opts).unwrap()
```

Comment:
why do we need this change? let and then unwrap?

Reply:
I picked up the syntax from some PR. I changed it to make it a single line code.